src/lib/stitching/exchange/v2/stitching.ts

```diff
@@ -357,21 +357,44 @@ export const exchangeStitchingEnvironment = ({
           fragment: gql`
             ... on CollectorResume {
              userId
+             purchases {
+                totalAuctionCount
+                totalPrivateSaleCount
+              }
             }
           `,
           resolve: async (parent, _args, context, info) => {
-            const exchangeArgs = {
-              buyerId: parent.userId,
-            }
+            const nonBnmoPurchases =
+              (parent?.purchases?.totalAuctionCount ?? 0) +
+              (parent?.purchases?.totalPrivateSaleCount ?? 0)
 
-            return await info.mergeInfo.delegateToSchema({
-              schema: exchangeSchema,
-              operation: "query",
-              fieldName: "commerceBuyerActivity",
-              args: exchangeArgs,
-              context,
-              info,
-            })
+            try {
+              const exchangeArgs = {
+                buyerId: parent.userId,
+              }
+
+              const bnmoPurchases = await info.mergeInfo.delegateToSchema({
+                schema: exchangeSchema,
+                operation: "query",
+                fieldName: "commerceBuyerActivity",
+                args: exchangeArgs,
+                context,
+                info,
+              })
+
+              return {
+                totalPurchases: bnmoPurchases.totalPurchases + nonBnmoPurchases,
```

Comment:
**praise:** I like how easy it is to understand the types of purchases are included in `totalPurchases` when reading this method.